lib/spec/album_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '.././music/music_album'
+require 'json'
+
+describe MusicAlbum do
+  context 'When testing creation of Album instance' do
+    it 'Should create a new person when we call the class constructor' do
+      new_album = MusicAlbum.new(true, 2)
+      expect(new_album.on_spotify).to eq true
+      expect(new_album.published_date).to eq 2
+    end
+  end
+  context 'When testing the Album methods' do
```

Comment:
Great Job with all unit testing 👍 As optional, remember that the last reviewer asks us to add more descriptions and separate the creation of the class from the methods.

Reply:
Thanks! I'll listen to your feedback